docs/_articles/en/user-guide/byotemplate.md

```diff
@@ -4,79 +4,40 @@ lang: en
-
-**Note:** Only updates made to the files listed here show up in source files in VS Code. There’s no such restriction when you use the CLI to specify templates for metadata objects.
-
-| Template Folder        | Default Template Files           |
-| ------------- |:-------------:|
-| apexclass     | DefaultApexClass.cls <br>_class.cls-meta.xml
-|apextrigger    | ApexTrigger.trigger <br> _trigger.trigger-meta.xml
-|Lightningapp   |DefaultLightningApp.app <br> DefaultLightningController.js <br> DefaultLightningCss.css <br> DefaultLightningHelper.js <br> DefaultLightningRenderer.js <br> DefaultLightningSVG.svg <br> DefaultLightningAuradoc.auradoc <br>auradefinitionbundle.app-meta.xml
-|Lightningcomponent |aura/default/default.cmp <br> aura/default/default.cmp-meta.xml <br> aura/default/defaultController.js <br> aura/default/defaultHelper.js <br> aura/default/defaultRenderer.js <br> aura/default/default.svg <br> aura/default/default.design <br> aura/default/default.css <br> aura/default/default.auradoc <br> lwc/default/default.js<br> lwc/default/default.html <br> lwc/default/default.js-meta.xml |
-|Lightningevent | defaultLightningEvt.evt <br> auradefinitionbundle.evt-meta.xml
-|Lightninginterface | DefaultLightningIntf.intf <br> auradefinitionbundle.intf-meta.xml
-|Lightningtest | DefaultLightningTest.resource
-|Project | Any file in the project folder
-|Staticresource | empty.resource <br> empty.js <br> empty.json <br> empty.css <br> empty.txt <br> _staticresource.resource-meta.xml
-|visualforcecomponent| DefaultVFComponent.component <br> _component.component-meta.xml
-|visualforcepage | DefaultVFPage.page <br> _page.page-meta.xml
-
-## Clone the Repo
-See GitHub documentation for instructions on how to clone repos.
+**Note**:
+This [git repo subdirectory](https://github.com/forcedotcom/salesforcedx-templates/tree/main/src/templates) contains a collection of official Salesforce templates for metadata components. Only updates made to the files listed in this directory show up in source files in VS Code. You can clone this subdirectory, or store your custom templates locally. Keeping the same folder structure, just update relevant template files with your code. Remove the files that you don’t wish to override. The Salesforce Extensions require your templates to follow the exact folder structure and nomenclature as this repo. For example, your Apex class templates must be in a folder named ``apexclass`` in a file named ``DefaultApexClass.cls``. There’s no such restriction when you use the CLI to specify templates for metadata objects. You do not need the rest of the repo `salesforcedx-templates`.
```

Comment:
If you store your own custom templates locally, then you need to worry about keeping the same folder structure, yes? Because if you clone, I'm assuming the structure comes with it? 
And I'm confused by "only updates made to the files listed in this directory show up..." What if you create local custom templates? I think you need to break out using the repo vs. what you do if you want to go custom. Does that make sense? And it sounds like if you the CLI your templates can be wherever.

Reply:
Okay, I think I was lost until I viewed the page. I personally would decide what you need to say in the note - it's really too long. And pull any of the relevant specifics to the sub topics underneath.